### PR TITLE
pscanrules: ignore RateLimit headers in Timestamp Disclosure

### PR DESCRIPTION
**Pull request by jsoref**
Ignore:
* "RateLimit-Reset"
* "X-RateLimit-Reset"
* "X-Rate-Limit-Reset"

Fixes zaproxy/zaproxy#7747

### CODE REVIEW

**Comment by kingthorin** [comment]
There’s a few unit tests that need updating.

**Review by kingthorin — APPROVED** [review]
Thanks

**Comment by thc202** [comment]
This needs to be rebased.

File: addOns/pscanrules/CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Maintenance changes.
 - The Salvation2 library used by the CSP scan rule was upgraded to v3.0.1. Alerts may now have an alert condition if the policy contains characters outside the accepted set.
 - The CSP scan rule now includes handling for policies defined in META tags, as well as two new alerts pertaining to those policies (Issue 7303).
+- Timestamp Disclosure scan rule now excludes values in "RateLimit-Reset", "X-RateLimit-Reset", and "X-Rate-Limit-Reset" headers (Issue 7747).
```
Comment (thc202) [inline addOns/pscanrules/CHANGELOG.md:23]: This needs to be moved to the `Unreleased` section.

**Review by kingthorin — APPROVED** [review]

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Pull request merged**